WalletConnectSharp.Core/Models/JsonRpcRequestMethods.cs

```diff
@@ -0,0 +1,44 @@
+namespace WalletConnectSharp.Core.Models;
+
+public static class JsonRpcRequestMethods
```

Comment:
Can we move these static fields to `JsonRpcRequest` and move `ValidJsonRpcRequestMethods` to `JsonRpcRequestMethods`

This would look like `JsonRpcRequest.IsWalletConnectMethod(request.Method)` or `JsonRpcRequest.IsSigningMethod(request.Method)`

And then accessing all the implemented method strings would be `JsonRpcRequestMethods.EthSign` 

We can keep `JsonRpcRequestMethod` (singular) as is